packs/kubernetes/actions/db_rds_spec.py

```diff
@@ -0,0 +1,39 @@
+import re
+import uuid
+from st2actions.runners.pythonrunner import Action
+
+
+class DatabaseRdsSpec(Action):
+    def run(self, payload):
+        # For dbidentifier
+        db_name = re.sub('[^0-9a-zA-Z]+', '-', payload['name']) + "-" + payload['namespace']
```

Comment:
Just a note about python dicts, the [] notation throws a KeyError when key does not exist. For example, payload['name'] would throw. Sometimes you can do payload.get('name', default_name) or wrap the lines of code in try catch block.